src/main/java/ru/hogwarts/school/repository/FacultyRepository.java

```diff
@@ -4,11 +4,14 @@
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 import ru.hogwarts.school.model.Faculty;
+import ru.hogwarts.school.model.Student;
 
 import java.util.Collection;
 
 @Repository
 public interface FacultyRepository extends JpaRepository<Faculty, Long> {
-    @Query("SELECT f FROM Faculty AS f WHERE f.color = ?1")
-    Collection<Faculty> findByColor(String color);
+    Collection<Faculty> findByColorIgnoreCase(String color);
+    Collection<Faculty> findByNameIgnoreCase(String name);
```

Comment:
Эти методы можно объединить:
```suggestion
    Collection<Faculty> findByColorIgnoreCaseOrNameIgnoreCase((String color);
```